tests/behat/features/ra_candidate2.feature

```diff
@@ -0,0 +1,30 @@
+Feature: A RAA manages ra candidates in the ra environment
+  In order to promote candidates
+  As a RAA
+  I must be able to promote and demote identities were I'm allowed to through the authorization config
+
+  Scenario: Provision an institution and a user to promote later on by an authorized institution
+    # Institution A
```

Comment:
It seems your comments mention a scenario, but I'm not able to read more about this scenario. Ideas:

1. Add that scenario (or a link to it) to this feature
2. Update the comments to match the institutions that are actually being configured

Reply:
Removed the comments, because this is the scenario.